### ISSUE TITLE
Request R package `ottr` v1.3.0 for class PS 3

### ISSUE THREAD

**Issue by dbroockman** [berkeley-dsep-infra/hubs#4008]
### Package Name

ottr v1.3.0

### Hub details

- Hub: r.datahub
- Language: R

### Course Name

PS 3, Professor David Broockman

### Semester Details

Spring 2023

### Installation Deadline

01/09

**Comment by dbroockman** [comment]
I have an RA who is QA'ing my assignments before the semester starts - any chance I could get this package update a bit earlier?

**Comment by balajialg** [comment]
Thanks @dbroockman for the nudge. We have created a PR (#4069) to make this version change and it should get merged into the hubs soon. Will give you an update once it is done. Thx

**Comment by balajialg** [comment]
@dbroockman The changes were merged to production 30 minutes ago. You can test your notebooks in an hour and do let us know if you face any issues?

**Comment by balajialg** [comment]
Closing this issue as package upgrade is successful!
![image](https://user-images.githubusercontent.com/2306166/212181360-ebc404b7-92cf-461b-a85c-e459711384f5.png)
